helm-frontend/public/config.js

```diff
@@ -1,4 +1,4 @@
 window.RELEASE = "v1.0.0";
 window.BENCHMARK_OUTPUT_BASE_URL =
-  "https://storage.googleapis.com/crfm-helm-public/lite/benchmark_output/";
+	"https://storage.googleapis.com/crfm-helm-public/lite/benchmark_output/";
```

Comment:
Lol somehow we have dueling linters which keep changing the whitespace back and forth.

Reply:
Yeah should probably resolve this as this has showed up in a few prs already